LearnWithMentor/Controllers/TaskController.cs

```diff
@@ -41,21 +43,22 @@ public TaskController(ITaskService taskService, IMessageService messageService,
         /// </summary>
         [HttpGet]
         [Route("api/task")]
-        public async Task<HttpResponseMessage> GetAllTasksAsync()
+        public async Task<ActionResult> GetAllTasksAsync()
         {
             try
             {
                 var allTasks = await taskService.GetAllTasksAsync();
                 if (allTasks != null)
                 {
-                    return Request.CreateResponse(HttpStatusCode.OK, allTasks);
+                    return Ok(allTasks);
                 }
-                return Request.CreateErrorResponse(HttpStatusCode.NoContent, "There are no tasks in database.");
+
+                return NoContent();
             }
             catch (Exception e)
             {
-
-                return Request.CreateErrorResponse(HttpStatusCode.InternalServerError, e);
+                ////tracer.Error(Request, ControllerContext.ControllerDescriptor.ControllerType.FullName, e);
```

Comment:
Why did you decide to leave this commented code?

Reply:
to know where put future logger

Reply:
It's better to replace with ```//TODO: Log error here```
```suggestion
                //TODO: Log error here
```